src/ads/components/ads-label.js

```diff
@@ -1,7 +1,8 @@
-vjs.AdsLabel = vjs.Component.extend({
+var vjsComponent = videojs.getComponent('Component');
+var AdsLabel = videojs.extend(vjsComponent, {
   /** @constructor */
-  init: function (player, options) {
-    vjs.Component.call(this, player, options);
+  constructor: function (player, options) {
+    vjsComponent.call(this, player, options);
 
```

Comment:
@wmcmurray The think is that we need to support 4.x for a few months still.
We could create a VJS5 branch and try to keep them both updated until we find a better solution.
Would that work for you?